cartography/intel/aws/cloudwatch.py

```diff
@@ -0,0 +1,195 @@
+import time
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_cloudwatch_alarm(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('cloudwatch', region_name=region)
+    response = client.describe_alarms()
```

Comment:
`describe_alarms` supports pagination. We should implement pagination here.

Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudwatch.html#CloudWatch.Paginator.DescribeAlarms